assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -30,6 +30,16 @@ export default Ember.Controller.extend({
   sortBy: ["last_run_at:desc"],
   sortedQueries: Ember.computed.sort("model", "sortBy"),
 
+  @computed
+  acceptedImportFileTypes() {
+    return [
```

Comment:
I am just wondering why we need all these variants of the JSON type? Do different browsers send different things? Usually it is application/json

Reply:
I don't know initial intentions behind this list, I've just moved it from other place in this PR. But looks like you're right and it's more than enough to have just `application/json`. For example, [text/javascript is obsolete, and application/x-javascript was experimental ](https://stackoverflow.com/a/9664327/3206146).

Reply:
And [the mime type for json text is application/json](https://stackoverflow.com/a/477819/3206146).

We also already use just `application/json` with file inputs in several places, it seems to work fine.